src/packagers/RawPackager.js

```diff
@@ -12,6 +13,12 @@ class RawPackager extends Packager {
       contents = await fs.readFile(contents ? contents.path : asset.name);
     }
 
+    // Create output path if not exist
+    const dir = path.dirname(this.bundle.name);
+    if (!await fs.exists(dir)) {
```

Comment:
I think you can directly do `fs.mkdirp`, `mkdirp` will internally check if the directory exists. It will save a few I/O calls

Reply:
Thanks, I'll fix that ASAP